src/alfasim_sdk/constants.py

```diff
@@ -0,0 +1,361 @@
+        Employs the equilibrium slip velocity model and the Mills (1985) effective dynamic viscosity expression.
+
+    - Santamaria2010Equilibrium - Santamaría-Holek (2010):
+        This model is more appropriate to use when the solid phase has properties similar to or equal to hydrate.
+        It was fitted by Qin et al. (2018) for hydrates.
+
+    - Thomas1965Equilibrium - Thomas (1965):
+        Employs the equilibrium slip velocity model and the Thomas (1965) effective dynamic viscosity expression.
+    """
+
+    NoModel = "no_model"
+    Thomas1965Equilibrium = "thomas1965_equilibrium"
+    Mills1985Equilibrium = "mills1985_equilibrium"
+    Santamaria2010Equilibrium = "santamaria2010_equilibrium"
+
+
+class NodeCellType(Enum):
+    Internal = "internal_node"
+    MassSource = "mass_source_boundary"
+    Pressure = "pressure_boundary"
+    Separator = "separator_node"
+
+
+class MassInflowSplitType(Enum):
+    """
+    PvtUserGorWc is currently only used for GenKey
+    """
+
+    ConstantVolumeFraction = "mass_inflow_split_type_constant_volume_fraction"
+    ConstantMassFraction = "mass_inflow_split_type_constant_mass_fraction"
+    Pvt = "mass_inflow_split_type_pvt"
+    PvtUserGorWc = "mass_inflow_split_type_pvt_user_gor_wc"
+    PvtUserGlrWc = "mass_inflow_split_type_pvt_user_glr_wc"
+
+
+class WellIndexPhaseType(Enum):
+    Gas = "well_index_phase_gas"
+    Oil = "well_index_phase_oil"
+    Water = "well_index_phase_water"
+    Liquid = "well_index_phase_liquid"
+
+
+class MassSourceType(Enum):
+    MassFlowRates = "mass_source_type_mass_flow_rates"
+    AllVolumetricFlowRates = "mass_source_type_all_volumetric_flow_rates"
+    FlowRateOilGorWc = "mass_source_type_flow_rate_oil_gor_wc"
+    FlowRateGasGorWc = "mass_source_type_flow_rate_gas_gor_wc"
+    FlowRateWaterGorWc = "mass_source_type_flow_rate_water_gor_wc"
+    TotalMassFlowRatePvtSplit = "mass_source_type_total_mass_flow_rate_pvt_split"
+
+
+class PipeThermalModelType(Enum):
+    AdiabaticWalls = "adiabatic_walls"
+    SteadyState = "steady_state_heat_transfer"
+    Transient = "transient_heat_transfer"
+
+
+class PipeThermalPositionInput(Enum):
+    Tvd = "position_by_tvd"
+    Md = "position_by_md"
+
+
+class PipeEnvironmentHeatTransferCoefficientModelType(Enum):
+    WallsAndEnvironment = "walls_and_environment_heat_transfer_coefficient"
+    WallsAndWater = "walls_and_water_heat_transfer_coefficient_model"
+    WallsAndAir = "walls_and_air_heat_transfer_coefficient_model"
+    Overall = "overall_heat_transfer_coefficient_model"
+
+
+class PVTCompositionalViscosityModel(Enum):
+    CorrespondingStatesPrinciple = "corresponding_states_principle"
+    LohrenzBrayClark = "lohrenz_bray_clark"
+
+
+class MaterialType(Enum):
+    Solid = "solid"
+    Fluid = "fluid"
+
+
+FLUID_GAS = "gas"
+FLUID_OIL = "oil"
+FLUID_WATER = "water"
+FLUID_DROPLET = "droplet"
+FLUID_BUBBLE = "bubble"
+FLUID_PHASE_NAMES = [FLUID_GAS, FLUID_OIL, FLUID_WATER, FLUID_DROPLET, FLUID_BUBBLE]
```

Comment:
Did you really copy this from alfasim?
I ask because as far I remember and not phases but dispersed fields.
Maybe before integrating this we should iron this out.

Maybe @arthursoprana can suggest someone to help review/update this.

Reply:
Indeed I'd prefer to simply have `FLUID_PHASE_NAMES = [FLUID_GAS, FLUID_OIL, FLUID_WATER]`.

I did a quick search at the code and found that `FLUID_DROPLET` and `FLUID_BUBBLE` are ONLY used in tests, so we probably should remove them from `alfasim_sdk` and leave it in `alfasim` (perhaps with a big warning msg that is tests only).

Reply:
Commit https://github.com/ESSS/alfasim-sdk/pull/108/commits/0bc25f3c62c277399a56a204feb06c71971c6156 remove `FLUID_DROPLET` and `FLUID_BUBBLE ` from `constants.py`